include/d/actor/d_a_steam_tag.h

```diff
@@ -3,29 +3,47 @@
 
 #include "f_op/f_op_actor.h"
 #include "d/d_cc_d.h"
+#include "d/d_particle.h"
 #include "d/d_point_wind.h"
+#include "f_op/f_op_actor_mng.h"
 
 class JPABaseEmitter;
 
+struct daSteamTag_mData {
+    /* 0x00 */ s16 field_0x00;
+    /* 0x02 */ s16 field_0x02;
+    /* 0x04 */ s16 mCreateTimer_seed;
+    /* 0x06 */ s16 mCreateTimer_offset;
+    /* 0x08 */ s16 mEmitTimer_offset;
+    /* 0x0A */ s16 mEmitTimer_seed;
+    /* 0x0C */ u8  mSteamAlpha;
```

Comment:
The word "seed" isn't right here, `cM_rndF`'s argument affects the maximum range of values it can return, not a random seed. e.g. `cM_rndF(5.0f)` -> anywhere in the range of 0.0f to 5.0f randomly. (I don't know why we have that parameter simply called "f" in c_math.cpp, but in c_math.h it's more clearly called "max".)

(Also, this isn't a hard rule, but I think the devs tended to generally use `mThisNamingStyle` for classes and `this_naming_style` for structs?)

So I would name these more along these lines:
```suggestion
    /* 0x04 */ s16 create_time_range;
    /* 0x06 */ s16 create_time_min;
    /* 0x08 */ s16 emit_time_min;
    /* 0x0A */ s16 emit_time_range;
    /* 0x0C */ u8  steam_alpha;
```